src/Partials/Components/datetime.jsx

```diff
@@ -10,10 +10,36 @@ module.exports = React.createClass({
   displayName: 'Datetime',
   mixins: [valueMixin, multiMixin],
   getInitialValue: function() {
-    return null;
+    if (
+      typeof this.props.component === "undefined" ||
+      typeof this.props.component.defaultDate === "undefined" ||
+      this.props.component.defaultDate.length === 0
+    ) {
+      return '';
+    }
+    const { defaultDate } = this.props.component;
+
+    var dateVal = new Date(defaultDate);
+    if (isNaN(dateVal.getDate())) {
+      try {
+        dateVal = new Date(eval(defaultDate));
```

Comment:
Well, I'm pretty sure it is available to the eval function, it's being imported at the top of the file.

It's also like that in the Angular version.
https://github.com/formio/ngFormio/blob/master/src/components/datetime.js#L29